app/javascript/src/NewService/components/NewServiceForm.jsx

```diff
@@ -10,12 +10,14 @@ type Props = {
   isServiceDiscoveryUsable: boolean,
   serviceDiscoveryAuthenticateUrl: string,
   providerAdminServiceDiscoveryServicesPath: string,
-  adminServicesPath: string
+  adminServicesPath: string,
+  apiap: boolean,
+  backendApis: Array<{id: number, name: string}>
```

Comment:
I think this prop should be optional:
```suggestion
  backendApis?: Array<{id: number, name: string}>
```